### PR TITLE
Add indonesian store gramedia

### PR DESCRIPTION
**Pull request by irhamdz**
add some bookstore in indonesian

### CODE REVIEW

File: brands/shop/books.json [hunk 1]
```diff
@@ -121,6 +121,20 @@
...
+    "countryCodes": ["id"],
+    "tags": {
+      "brand": "Gramedia",
+      "brand:en": "Gramedia",
+      "brand:id": "Gramedia (toko buku)",
+      "brand:wikidata": "Q19646130",
+      "brand:wikipedia": "id:Gramedia (toko buku)",
+      "name": "Gramedia",
+      "name:en": "Gramedia",
+      "name:id": "Gramedia (toko buku)",
```
Comment (bhousel) [inline brands/shop/books.json]: Does it really need `brand:en`, `brand:id`, `name:en` and `name:id` tags?   I don't understand them..
It looks like this brand is just "Gramedia" always?

Comment (irhamdz) [reply brands/shop/books.json]: ah my bad, i will fix it

**Comment by irhamdz** [comment]
i already update it, maybe you can check it @bhousel

**Comment by bhousel** [comment]
Thanks @irhamdz looks great ! 👍

**Pull request merged**